.pipelines/pipeline.yaml

```diff
@@ -330,25 +336,45 @@ stages:
       clusterName: "swifte2e"
       osSku: "Ubuntu"
 
-  - template: singletenancy/aks-engine/e2e-job-template.yaml
+  - template: singletenancy/aks/e2e-job-template.yaml
     parameters:
       name: "ubuntu_18_04_linux_e2e"
-      displayName: Ubuntu 18.04
-      pipelineBuildImage: "$(BUILD_IMAGE)"
-      clusterDefinition: "cniLinux1804.json"
-      clusterDefinitionCniTypeKey: "azureCNIURLLinux"
-      clusterDefinitionCniBuildOS: "linux"
-      clusterDefinitionCniBuildExt: ".tgz"
+      displayName: AKS Ubuntu 18.04
+      arch: amd64
+      os: linux
+      clusterType: linux-cniv1-up
+      clusterName: "ubuntu18e2e"
+      k8sVersion: 1.24.9
```

Comment:
Does it not default to 1.24.x? I do not think that we should specify versions as they will get outdated. 

Also 1.24.x is End of Life this month. Should we consider shifting this to ubuntu 22 and 1.25.x+?

Reply:
No by default it is greater than 1.25, the reason for this is aks does not support ubuntu 18 with k8s 1.25.
The pipeline currently is testing ubuntu 18 so we need to test for ubuntu 18.
https://learn.microsoft.com/en-us/azure/aks/cluster-configuration

Regarding the ubuntu 22, you are right we need to support that. Once this is merged. It would be just a matter of adding another set of parameters for Ubuntu 22.